services/report/3rdparty/jasper-cs-report/src/main/resources/TombstoneWithImages.jrxml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.17.0.final using JasperReports Library version 6.17.0-6d93193241dd8cc42629e188b94f9e0bc5722efd  -->
+<jasperReport xmlns="http://jasperreports.sourceforge.net/jasperreports" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jasperreports.sourceforge.net/jasperreports http://jasperreports.sourceforge.net/xsd/jasperreport.xsd" name="GroupWithImages" pageWidth="612" pageHeight="792" columnWidth="540" leftMargin="36" rightMargin="36" topMargin="0" bottomMargin="18" uuid="6ff7d5f1-1ce4-458e-bc9c-a91b39731abf">
+	<property name="com.jaspersoft.studio.data.defaultdataadapter" value="Tunnel to CSpace Prod"/>
+	<parameter name="csid" class="java.lang.String" isForPrompting="false">
+		<parameterDescription><![CDATA[Object CSID]]></parameterDescription>
+	</parameter>
+	<parameter name="groupcsid" class="java.lang.String" isForPrompting="false">
+		<parameterDescription><![CDATA[Group CSID]]></parameterDescription>
+	</parameter>
+	<parameter name="csidlist" class="java.lang.String" isForPrompting="false">
+		<parameterDescription><![CDATA[Group CSID]]></parameterDescription>
+	</parameter>
+	<parameter name="objcsids" class="java.lang.String" isForPrompting="false">
+		<defaultValueExpression><![CDATA[$P{csidlist} != null ? ("'" + $P{csidlist}.replaceAll(",", "','") + "'") : "'NOVALUE'"]]></defaultValueExpression>
+	</parameter>
+	<queryString>
+		<![CDATA[SELECT
+  cc.objectnumber AS idNumber,
+  tg.title AS title,
+  ng.objectname AS objectName,
+  regexp_replace(pp.objectproductionperson, '^.*\)''(.*)''$', '\1') AS prodPerson,
+  regexp_replace(po.objectproductionorganization, '^.*\)''(.*)''$', '\1') AS prodOrg,
+  pd.datedisplaydate AS prodDate,
+  mpg.dimensionsummary AS dimensionsSum,
+  cc.editionnumber AS editionNum,
+  regexp_replace(nc.item, '^.*\)''(.*)''$', '\1') AS namedCollection,
+  regexp_replace(cc.computedcurrentlocation, '^.*\)''(.*)''$', '\1') AS compCurrLocation,
+  (SELECT 'https://core-dev.cspace.berkeley.edu/cspace-services/blobs/' || mc.blobcsid || '/derivatives/Small/content'
```

Comment:
Lam, does this work? Wouldn't the request need to be authenticated to be able to retrieve the image through the REST API?